src/deluge/model/mod_controllable/mod_controllable_audio.cpp

```diff
@@ -1796,9 +1796,12 @@ void ModControllableAudio::receivedCCFromMidiFollow(ModelStack* modelStack, Clip
 				if (midiFollow.paramToCC[xDisplay][yDisplay] == ccNumber) {
 					// obtain the model stack for the parameter the ccNumber received is learned to
 					// don't display "can't control param" error message if you're in a MIDI or CV clip
-					bool displayError = midiEngine.midiFollowDisplayParam
-					                    && (clip->output->type != OutputType::MIDI_OUT)
-					                    && (clip->output->type != OutputType::CV);
+					bool isMIDIORCV = false;
+					if (clip) {
+						isMIDIORCV =
```

Comment:
I think this only actually applies to CV, midi shouldn't fail and should show an error if it does